src/ClockPanel.tsx

```diff
@@ -247,14 +247,14 @@ class UnthemedClockPanel extends PureComponent<Props, State> {
     }
 
     return (
-      <h4 className={className}>
+      <span className={className}>
```

Comment:
Even though this might (for now) fix the centering issue we will be back to the same situation that brought this problem: relying on the page base styles that could change over updates.

A better solution is to keep the original h4 tag and write the necessary CSS to remove the margin in the element [here](https://github.com/grafana/clock-panel/blob/master/src/ClockPanel.tsx#L223)

Reply:
Sure, I've added `magin: 0` to these elements.